packages/flutter/lib/src/widgets/scroll_context.dart

```diff
@@ -41,6 +50,10 @@ abstract class ScrollContext {
   /// The direction in which the widget scrolls.
   AxisDirection get axisDirection;
 
+  /// Platform specific function applied to existing velocity during repeated
+  /// scrolls.
+  MomentumCarryFunction get momentumCarryFunction;
```

Comment:
Ya, good idea here and below. Moved the stuff out of the context and consolidated in the physics